protos/dcs/common/v0/common.proto

```diff
@@ -360,13 +360,22 @@ message Target {
  * players but can also be used by things like Jester for setting waypoints
  */
 message MarkPanel {
+  // The id of the mark panel.
   uint32 id = 1;
-  // I have no idea what this time value means
+  // The time in seconds relative to the mission start the mark got created.
   double time = 2;
-  Unit initiator = 3;
+  // The unit of the player that created the mark. Not set if the player isn't
+  // controlling any unit anymore (disconnected, spectator, game master, ...).
+  optional Unit initiator = 3;
+  // If set, the mark is only visible for the specified coalition.
   optional Coalition coalition = 4;
+  // The ID of the group the player was in when creating the mark panel. This
+  // will still be set even if the player isn't controlling the unit in that
+  // group anymore.
   optional uint32 group_id = 5;
```

Comment:
I kept `group_id` in while revising the PR. Though I am still not sure if this is a useful propert as it isn't possible to retrieve a group by id.

Reply:
Maddeningly the `OutTextToGroup` API does in fact use the group ID while almost everything else uses the name...

So I think keeping it is fine.